integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXImagesTests.java

```diff
@@ -96,7 +96,8 @@ public void testDistroXWithBaseImageCanBeCreatedSuccessfully(TestContext testCon
                 .when(distroXTestClient.create(), key(distrox))
                 .await(STACK_AVAILABLE)
                 .then((tc, testDto, client) -> {
-                    return waitUtil.waitForDistroxInstancesStatus(testDto, client, instancesHealthy);
+                    waitUtil.waitForDistroxInstanceStatus(testDto.getResponse().getName(), tc, instancesHealthy);
```

Comment:
I don't really like the Idea that the return type of this method was changed to void.
Breaks the flow in the testcase.

Reply:
I could revert this change only for here (for waitForDistroxInstancesStatus method). In case of waitForSdxInstancesStatus this was part of the required change.